deepsampler-provider/src/testFixtures/java/de/ppi/deepsampler/provider/common/PersistentSamplerAspectTest.java

```diff
@@ -544,4 +544,40 @@ private PersistentSampleManager save(Path pathToFile) {
         return source;
     }
 
+    @Test
+    public void polymorphicSamplesCanBeRecordedAndLoaded() throws IOException {
+        // Ensure, that no Samples from previous tests exists...
+        Sampler.clear();
+
+        // GIVEN
+        final TestService testServiceSampler = Sampler.prepare(TestService.class);
+        PersistentSample.of(testServiceSampler.getAnimal());
+
+        // make the method call that is recorded
+        getTestService().getAnimal();
+
+        // save the recorded sample to file...
+        final String pathToFile = "./record/polymorphicSamplesCanBeRecordedAndLoaded.json";
+        final PersistentSampleManager source = PersistentSampler.source(JsonSourceManager.builder().buildWithFile(pathToFile));
+        source.record();
+
+        // Reset SampleRepository to ensure that only deserialized samples will be used in the next steps...
```

Comment:
This is copied code. there is a method for that: https://github.com/ppi-ag/deep-sampler/blob/cb9876917ac97d3cc270ba69b3e2e5ee0f39047d/deepsampler-provider/src/testFixtures/java/de/ppi/deepsampler/provider/common/PersistentSamplerAspectTest.java#L535